### PR TITLE
Fastpass executable in source code

### PR DESCRIPTION
**Pull request by tpasternak**

### CODE REVIEW

File: fastpass/bin/fastpass [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+exec coursier launch org.scalameta:fastpass_2.12:1.6.0 --main scala.meta.fastpass.Fastpass -- $@
```
Comment (olafurpg) [inline fastpass/bin/fastpass]: Can we add a sibling `coursier` script and use that to bootstrap?

```
curl -fLo coursier https://git.io/coursier-cli &&
    chmod +x coursier &&
    ./coursier
```

```suggestion
COURSIER="$(dirname $0)/coursier"
exec $COURSIER launch fastpass --contrib -- "$@"
```

**Review by olafurpg — CHANGES_REQUESTED** [review]
Thank you for this contribution! A small suggestion so this script is self-contained

File: fastpass/bin/fastpass (hunk 1 above)
Comment (tpasternak) [reply fastpass/bin/fastpass]: we'd have to manually update coursier, too, when new versions are released

Comment (tpasternak) [reply fastpass/bin/fastpass]: I don't say, it's a bad idea, I'm just wondering

Comment (olafurpg) [reply fastpass/bin/fastpass]: I don't think we would have to manually update coursier, it should continue to work no matter what version of fastpass we're using

Comment (olafurpg) [reply fastpass/bin/fastpass]: The `coursier` slim binary should be committed into the repo

Comment (tpasternak) [reply fastpass/bin/fastpass]: ok, done

**Review by olafurpg — APPROVED** [review]

**Pull request merged**